File: Identity.Application/DTOs/UserDto.cs
using Core.Domain.ValueObjects;
using Core.Shared.DTOs.HR;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Identity.Application.DTOs
{
    public class UserDto
    {

        public Guid Id { get;  set; }
        public string? UserName { get;  set; }
        public string? phoneNumber { get;  set; }
        public string? NickName { get; set; }
        public string? Email { get;  set; }
        public PersonDto? person { get; set; } = null;
        //public ICollection<RoleDto>? roles { get; set; } = null;
        public ICollection<string>? roles { get; set; } = null;
    }
}

File: FrontEnd/src/modules/Identity/models/UserDto.ts
import { PersonDto } from "@/modules/HR/models/PersonDto";

// modules/identity/models/UserDto.ts
export interface UserDto {
  userName: string;
  id: string ;
  phoneNumber: string | null;
  nickName?: string | null;
  email?: string | null;
  person?: PersonDto | null;
}
